Huion_H640P/index.md

```diff
@@ -0,0 +1,23 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: H640P
+image: Huion_H640P
+report_rate: 233
+working_area:
+    width: 6.3
+    height: 3.9
+    resolution: 5080
+pen:
+    pressure_levels: 8192
+frame_controls: 6 buttons
+sold_as:
+    - Huion H640P
+    - Huion Inspiroy H640P Graphics Drawing Tablet
+supported: true
+supported_in:
+    kernel: ">= 3.17"
+    digimend: ">= 5"
```

Comment:
It's supported neither in the kernel nor in the DIGImend drivers yet, please remove these two lines, and please set `supported` to `false`.